cbv/templates/base.html

```diff
@@ -8,6 +8,10 @@
     <meta name="author" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+    <!-- Favicon -->
+    <link rel="icon" href="{% static 'favicon/favicon.ico' %}" sizes="48x48" />
```

Comment:
`favicon.ico` is added as a backup for Safari. Recommended in `2. The .ico favicon (for Safari and IE)` section of https://medium.com/web-dev-survey-from-kyoto/favicon-nightmare-how-to-maintain-sanity-7628bfc39918

Other browsers pick up svg icon.